mirrord-operator/templates/role.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
why not add it to the existing role?

Reply:
It needs to be a role and not cluster role to be namespace specific (also I saw a very similar builtin role to do the same but not sure I can trust it always exists)

Reply:
ahh! ok